dask/dataframe/io/csv.py

```diff
@@ -321,8 +359,13 @@ def read_pandas(reader, urlpath, blocksize=AUTO_BLOCKSIZE, collection=True,
 
     header = b'' if header is None else parts[skiprows] + b_lineterminator
 
-    # Use sample to infer dtypes
+    # Use sample to infer dtypes and check for presense of include_path_column
     head = reader(BytesIO(b_sample), **kwargs)
+    if include_path_column and (include_path_column in head.columns):
+        raise KeyError("Files already contain the column name: %s, so the "
```

Comment:
Oop, sorry, missed this one. Why is this a `KeyError`? I'd expect a `ValueError` instead, as there's an issue with the value the user provided, and a different value could fix it.

Reply:
Whoops. I guess I was thinking column names are like keys. Fixed.